variables.tf

```diff
@@ -258,7 +258,7 @@ variable "upload_token_lifetime_mins" {
   default = "1440"
 }
 variable "metrics_config" {
-  default = "{ \"CONTACT_UPLOAD\": 60, \"CHECK_IN\": 60, \"FORGET\": 60, \"TOKEN_RENEWAL\": 60, \"CALLBACK_OPTIN\": 60, \"DAILY_ACTIVE_TRACE\": 60, \"CONTACT_NOTIFICATION\": 60 }"
+  default = "{ \"CONTACT_UPLOAD\": 60, \"CHECK_IN\": 60, \"FORGET\": 60, \"TOKEN_RENEWAL\": 60, \"CALLBACK_OPTIN\": 60, \"DAILY_ACTIVE_TRACE\": 60, \"CONTACT_NOTIFICATION\": 60, \"LOG_ERROR\": 1 }"
```

Comment:
@colmharte the rate limiting should be less than once per 60 seconds for this error I assume, but is once per second a sensible value?

Reply:
its once per 60 per person ? In which case thats ok